index.js

```diff
@@ -762,6 +765,35 @@ class GithubScm extends Scm {
             sha: pullRequestInfo.head.sha
         }));
     }
+
+    /**
+     * Get an array of scm context (e.g. github.com)
+     * @method getScmContexts
+     * @return {Array}
+     */
+    _getScmContexts() {
+        const contextName = this.config.gheHost
+            ? [`github:${this.config.gheHost}`]
+            : ['github.com'];
+
+        return contextName;
+    }
+
+    /**
+     * Determine if a scm module can handle the received webhook
+     * @method canHandleWebhook
+     * @param {Object}    headers    The request headers associated with the webhook payload
+     * @param {Object}    payload    The webhook payload received from the SCM service
+     * @return {Promise}
+     */
+    _canHandleWebhook(headers, payload) {
+        return this._parseHook(headers, payload)
+            .then((result) => {
+                Promise.resolve(result !== null);
+            }).catch(() => {
+                Promise.relolve(false);
```

Comment:
This then and catch clause seems to return nothing.
You need to fix it to return resolved result.